python/Ganga/Runtime/GPIexport.py

```diff
@@ -18,6 +18,31 @@
 from Ganga.GPIDev.Base.Objects import GangaObject
 from inspect import isclass
 
+def _addToInterface(interface, name, _object):
+
+    if isType(_object, GangaObject):
+        setattr(interface, name, addProxy(_object))
+    elif isclass(_object) and issubclass(_object, GangaObject):
+        setattr(interface, name, getProxyClass(_object))
+    else:
+        setattr(interface, name, _object)
+
+def exportToInterface(myInterface, name, _object, doc_section, docstring=None):
```

Comment:
Are there any situations where we want access to the `JobRegistrySliceProxy` object in *our* code? I assume not.

In what cases do we really want access to a `JobRegistrySlice` in our code? Is access to the `JobRegistry` not good enough?